toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceImpl.kt

```diff
@@ -90,6 +90,9 @@ class CratesLocalIndexServiceImpl
     @Volatile
     private var state: CratesLocalIndexState = CratesLocalIndexState()
 
+    /**
+     * [isReady] will be true when index is not updating
+     */
     private val isReady: AtomicBoolean = AtomicBoolean(true)
```

Comment:
I read this code again and name is still quite confusing. Maybe something like `isUpdating` is better here?